src/EnergyPlus/EMSManager.cc

```diff
@@ -2012,9 +2012,9 @@ namespace EMSManager {
 			if ( ! Surface( SurfNum ).HeatTransSurf ) continue;
 			if ( ! ( Surface( SurfNum ).ExtBoundCond == ExternalEnvironment ) ) continue;
 
-			SetupEMSActuator( "Surface", Surface( SurfNum ).Name, "Outdoor Air Dryblub Temperature", "[C]", Surface( SurfNum ).OutDryBulbTempEMSOverrideOn, Surface( SurfNum ).OutDryBulbTempEMSOverrideValue );
+			SetupEMSActuator( "Surface", Surface( SurfNum ).Name, "Outdoor Air Drybulb Temperature", "[C]", Surface( SurfNum ).OutDryBulbTempEMSOverrideOn, Surface( SurfNum ).OutDryBulbTempEMSOverrideValue );
```

Comment:
Well, dryblub isn't found in the idfs, so that's one less thing to do here.  Currently working on the transition changes, @mitchute.